samples/LibuvWithNonAllocatingFormatters/Program.cs

```diff
@@ -89,9 +90,15 @@ static void RunLoop(bool log)
                         formatter.Format(" @ {0:O}", DateTime.UtcNow);
                     }
 
-                    var response = formatter.Buffer.Slice(0, formatter.CommitedByteCount); // formatter should have a property for written bytes
-
-                    connection.TryWrite(response);
+                    unsafe
+                    {
+                        fixed (byte* ptr = formatter.Buffer)
```

Comment:
We could pass the pointer directly here but I just wanted to reduce the number of changes.